analyzer/src/main/kotlin/curation/NoCurationProvider.kt

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2020 Bosch.IO GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.analyzer.curation
+
+import org.ossreviewtoolkit.analyzer.PackageCurationProvider
+import org.ossreviewtoolkit.model.Identifier
+import org.ossreviewtoolkit.model.PackageCuration
+
+/**
+ * A [PackageCurationProvider] that does not provide any curations.
```

Comment:
Consider instead adding the below to PackageCurationProvider interface:
```
    companion object {
        val EMPTY = object : PackageCurationProvider {
            override fun getCurationsFor(pkgId: Identifier): List<PackageCuration> = emptyList()
        }
    }
```

Reply:
I was considering this initially but discarded it due to the verbose syntax until I realized this gives my a good excuse to try out the [new SAM conversion for Kotlin interfaces](https://kotlinlang.org/docs/reference/whatsnew14.html#sam-conversions-for-kotlin-interfaces) 😁

Reply:
Ah, I was looking for this just a few days before you upgraded to 1.4 without success - but wasn't aware that it's now available.